korman/exporter/convert.py

```diff
@@ -107,6 +107,11 @@ def run(self):
                 self.report.progress_end()
                 self.report.save()
 
+                # Step 5.2: If any nonfatal errors were occured during the export, we will
```

Comment:
The phrase "were occured" is strange (a combination of "were encountered" and "occurred" perhaps?).  Just "occurred" should be sufficient.